core/iwasm/interpreter/wasm_interp_fast.c

```diff
@@ -5650,6 +5652,25 @@ wasm_interp_call_func_bytecode(WASMModuleInstance *module,
                 GET_OPCODE();
 
                 switch (opcode) {
+                    // Memory
```

Comment:
Had better use `/* .. */` to add comment, we usually use it for C source files. Same for below.